ts/src/gate.ts

```diff
@@ -973,6 +953,18 @@ export default class gate extends Exchange {
         return super.safeMarket (marketId, market, delimiter, marketType);
     }
 
+    safeNetwork (networkId) {
```

Comment:
instead of safeNetwork, we should use `networkIdToCode` and `networkCodeToId`

Reply:
done, I've also remove some strange mappings

Reply:
@ndubel thanks will check it